perfkitbenchmarker/providers/gcp/gcp_spanner.py

```diff
@@ -109,13 +110,14 @@ def _Exists(self):
     cmd = util.GcloudCommand(self, 'spanner', 'databases', 'list')
     cmd.flags['filter'] = self._database
     cmd.flags['instance'] = self._name
+    # Suppress warning to prevent it from pollute stdout.
```

Comment:
it will not pollute stdout.
https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/perfkitbenchmarker/vm_util.py#L258